doc/hcr.rst

```diff
@@ -187,7 +188,25 @@ Native code targets
 Native projects using the hot code reloading option will be implicitly
 compiled with the `-d:useNimRtl` option and they will depend on both
 the ``nimrtl`` library and the ``nimhcr`` library which implements the
-hot code reloading run-time.
+hot code reloading run-time. Both libraries can be found in the ``lib``
+folder of Nim and can be compiled to a ``.dylib`` file to satisfy compiling
```

Comment:
Please note that `lib<name>.dylib` is the format for macOS. This differs between operating systems (`<name>.dll` on Windows and `lib<name>.so` on *nix).

I think we can just get away with "compiled into dynamic libraries".

> satisfy compiling of the example code above.

It's actually only needed at runtime.

Reply:
Thank you ! Absolutely right.

I've committed `6b31eb153e29ed670141689f71e70d8ef5aeba79` to resolve this.